cmd/gaia/init/validate_genesis.go

```diff
@@ -18,21 +18,23 @@ func ValidateGenesisCmd(ctx *server.Context, cdc *codec.Codec) *cobra.Command {
 		Args:  cobra.RangeArgs(0, 1),
 		Short: "validates the genesis file at the default location or at the location passed as an arg",
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
-			var genesis string
-			var genDoc types.GenesisDoc
-			var genstate app.GenesisState
 
 			// Load default if passed no args, otherwise load passed file
+			var genesis string
 			if len(args) == 0 {
 				genesis = ctx.Config.GenesisFile()
 			} else {
 				genesis = args[0]
 			}
 
+			fmt.Printf("validating genesis file at %s\n", genesis)
```

Comment:
This should go to stderr

Reply:
Why?

Reply:
According to [POSIX](http://pubs.opengroup.org/onlinepubs/009695399/) _"standard error is for writing diagnostic output; standard output is for writing conventional output"_. All CLI commands' logs should go to `/dev/stderr`.

Reply:
@jackzampolin bump

Reply:
I'm not particularly crazy about outputting to `stderr` in this situation TBH.